utils/rector/RewriteFactoryPhpDoc

```diff
@@ -0,0 +1,3 @@
+PHPUnit 10.5.13 by Sebastian Bergmann and contributors.
+
+Required argument for option "--filter" is missing
```

Comment:
It looks as if this was committed by mistake.

Reply:
wow nice spot, thanks :sweat_smile: